vonk/reference/programming_api/bundles.rst

```diff
@@ -0,0 +1,145 @@
+.. _vonk_reference_api_bundles:
+
+Constructing bundles
+====================
+
+In a Plugin or Facade you may need to construct a bundle from a set of resources, e.g. a SearchResult (see :ref:`here <vonk_reference_api_isearchrepository>`). There are two ways of doing this: with the Bundle POCO or with SourceNodes.
+
+Bundle POCO
+-----------
+
+This is fairly straightforward. Create a new Bundle object, and fill its properties, iterating over the set of resources that you have. The code looks like this:
```

Comment:
I thought Ewout wanted POCOs deprecated at some point? If that's the case, maybe we shouldn't promote it as the first choice?

Reply:
POCOs are not deprecated any time soon. But internally for our tooling - that often has to support multiple FHIR versions - we'd rather not use them. If you make a plugin that is only used for R4 - by all means use R4 POCO's.